packages/features/webhooks/lib/scheduleTrigger.ts

```diff
@@ -45,7 +45,10 @@ export async function addSubscription({
       },
     });
 
-    if (triggerEvent === WebhookTriggerEvents.MEETING_ENDED) {
+    if (
+      triggerEvent === WebhookTriggerEvents.MEETING_ENDED
+      // || triggerEvent === WebhookTriggerEvents.MEETING_STARTED
```

Comment:
not sure about this

Reply:
I think this is needed for zapier and make.com. As we mostly likely add that trigger there too one day, we can already add this here as well 👍